File: scalechain-net/src/test/scala/io/scalechain/blockchain/net/handler/GetHeadersMessageHandlerSpec.scala
package io.scalechain.blockchain.net.handler

import java.io.File

import io.netty.channel.embedded.EmbeddedChannel
import org.scalatest._


class GetHeadersMessageHandlerSpec extends MessageHandlerTestTrait with Matchers {
  this: Suite =>

  val testPath = new File("./target/unittests-GetHeadersMessageHandlerSpec/")

  "handle" should "" in {
  }
}

File: scalechain-net/src/test/kotlin/io/scalechain/blockchain/net/handler/GetHeadersMessageHandlerSpec.kt
package io.scalechain.blockchain.net.handler

import io.kotlintest.matchers.Matchers
import java.io.File

class GetHeadersMessageHandlerSpec : MessageHandlerTestTrait(), Matchers {

  override val testPath = File("./target/unittests-GetHeadersMessageHandlerSpec/")

  init {
    "handle" should "" {
    }
  }
}